proto/ligato/vpp/l3/vrrp.proto

```diff
@@ -4,11 +4,13 @@ package ligato.vpp.l3;
 
 option go_package = "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/l3;vpp_l3";
 
+// VRRPEntry represents Virtual Router desired state.
 message VRRPEntry {
     string interface = 1;
     uint32 vr_id = 2;
+    // Priority defines which router becomes master.
     uint32 priority = 3;
-    // VR advertisement interval.
+    // VR advertisement interval in centiseconds.
```

Comment:
Oh my, I dont believe I actually measured time in centiseconds like ever. I think we should choose something more common, if seconds are too big than just go with milliseconds and convert to "centiseconds" down the line if that's what VPP expects.

Reply:
Good point.

Reply:
This strange time unit is used in the RFC for VRRPv3 (VRRPv2 uses seconds): https://tools.ietf.org/html/rfc5798
It is probably the smallest time unit that a VRRP router must support.

Reply:
What is the best way to handle this situation? I am planing to make user put milliseconds and then covert to centiseconds, but some error will be present (for example user puts 55 and gets 50). Should I leave the centiseconds in this case?

Reply:
I would go for milliseconds, it's definitely more common and users wont need to think much about conversion from centiseconds. The small error wont matter for sure.

Reply:
I'm OK with milliseconds - maybe just log warning if (value % 10 != 0)

Reply:
Okay, then I will do the millisecond with option.